CMakeLists.txt

```diff
@@ -46,6 +46,7 @@ set(MODULES
   gpu
   gameboy
   debugger
+	auto_controller
```

Comment:
Indentation is off, configure your editor to show whitespace so you can change spaces/tabs here.